### PR TITLE
Add evan subdomain to dino.icu.yaml

### PR DESCRIPTION
**Pull request by evan-gan**
# Adding `evan.dino.icu`

## Description

Adding so I can more easily share projects hosted on a server I have setup

### CODE REVIEW

File: dino.icu.yaml [hunk 1]
```diff
@@ -172,6 +172,12 @@ erdtree: # erdtree.dino.icu - contact @justhypex on discord for issues/enquiries
   - exchange: mail.vortex.skyfall.dev.
     preference: 500
 
+evan:
+  ttl: 600
+  type: A
+  values:
+    - "129.213.147.149:25565"
```
Comment (alx-alexpark) [inline dino.icu.yaml:179]:
```suggestion
  value:
    - 129.213.147.149
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
Hii evan!!!!!!, you can't put a port in an A record, but I assume this is for a minecraft server, and just having the IP in the A record will work just fine!!

File: dino.icu.yaml (hunk 1 above)
Comment (evan-gan) [reply dino.icu.yaml:179]: Hi alex! It's actually not for a minecraft server, that just happens to be the only port open on the server.

Comment (evan-gan) [reply dino.icu.yaml:179]: Ok!

**Review by alx-alexpark — APPROVED** [review]
very sus :+1:

**Pull request merged**